core/src/main/java/de/sesu8642/feudaltactics/lib/gamestate/GameStateHelper.java

```diff
@@ -46,6 +46,7 @@ public static GameState getCopy(GameState original) {
         if (original.getWinner() != null) {
             result.setWinner(copiedPlayers.get(original.getPlayers().indexOf(original.getWinner())));
         }
+        result.setBotIntelligence(original.getBotIntelligence());
```

Comment:
Good catch! I wonder if this might have caused some issues before. But it seems like it was only used in the IngameScreen, EditorScreen and tests. All of those places seem to be uncritical.